tests/utils/test_storage_utils.py

```diff
@@ -24,7 +27,7 @@ class TestStorageUtils(TestCase):
     @mock.patch("os.makedirs")
     def test_write_output_csv(self, do_nothing, to_mock_csv):
         df = pd.read_csv("tests/test_data/test_triage_results.csv")
-        os.environ["BASE_TRIAGE_DIR"] = "test"
+        cc.BASE_TRIAGE_DIR = "test"
```

Comment:
Recommended approach is to use mocks for updating constants when testing. This ensures that the constants are resets to original value after execution of each test.

Reply:
done